layouts/download/list.html

```diff
@@ -1,11 +1,11 @@
 {{ define "main"}}
 
 <section id="features" class="features section downloads ">
-        <div class="container-fluid downloados downloadsize" style="margin-top: 50px;">
+        <div class="container downloados" style="margin-top: 50px;">
             <div class="row" style="padding: 0px !important;text-align: center;">
                 <div class="dow" style="display: inline-block; width: 80%;">
                     {{ range .Site.Data.tabs}}
-                    <div class="icon-item col-xl-4 col-lg-4 col-md-4 col-sm-4  col-xs-4 text-center" >
+                    <div class="icon-item col-xl-4 col-lg-4 col-md-4 col-sm-4  col-xs-14 text-center" >
```

Comment:
I know the original code was not beautiful, but please do not reference nonexistent classes.